run.sh

```diff
@@ -70,20 +86,20 @@ echo "$(date) $num metrics!"
 
 
 waitTimeBoundary 0 60
-targets 5min | head -n 3 | runTest "min-diversity" 5min 180s 200
+targets 5min | head -n 3 | runTest "min-diversity" 5min 180s $RATE_HIGH
 
 waitTimeBoundary 20 60
-targets 5min | runTest "max-diversity" 5min 180s 200
+targets 5min | runTest "max-diversity" 5min 180s $RATE_HIGH
 
 waitTimeBoundary 20 60
-targets 1h | head -n 3 | runTest "min-diversity" 1h 180s 100
+targets 1h | head -n 3 | runTest "min-diversity" 1h 180s $RATE_HIGH
 
 waitTimeBoundary 20 60
-targets 1h | runTest "max-diversity" 1h 180s 100
+targets 1h | runTest "max-diversity" 1h 180s $RATE_LOW
```

Comment:
all the tests are in pairs. once min-diversity then max-diversity. they should use the same rate.
you probably want the min-diversity one above to use the low rate, because 1h of data doesn't always fit in NMT.